docs: initial accessibility section in each component page (#175)
Refinitiv/refinitiv-ui: documents/src/pages/elements/datetime-picker.md

  </div>
</ef-datetime-picker>
```

## Accessibility
::a11y-intro::

`ef-datetime-picker` provides input field for users to entering date string value or date with time value. Users can open popup with calendar element and use keyboard navigation to select the date from UI.

> `ef-datetime-picker` is assigned with role and implemented keyboard navigation. However, you must ensure that the element has associated label by using`aria-label` or `aria-labelledby`.

Reviewer: ```suggestion
`ef-datetime-picker` is implemented keyboard navigation for users to navigate on UI. You must ensure that the element has associated label by using `aria-label` or `aria-labelledby`.
```
